scripts/database/db-explore.sh

```
@@ -0,0 +1,158 @@
+    exit 1
+fi
+
+# Show database info before launching
+echo -e "${CYAN}📊 Database Explorer - Digests API${NC}"
+echo -e "${GRAY}Container: $CONTAINER_NAME${NC}"
+echo -e "${GRAY}Database: $DB_PATH${NC}"
+echo ""
+
+# Get table info
+echo -e "${GREEN}📋 Available Tables:${NC}"
+docker exec "$CONTAINER_NAME" sqlite3 "$DB_PATH" ".tables" | tr ' ' '\n' | sort | while read table; do
```

Shellcheck (0.10.0)

[info] read without -r will mangle backslashes.

(SC2162)